engines/router/missionctl/handlers/logutils.go

```diff
@@ -133,3 +142,8 @@ func uncompressHTTPBody(header http.Header, body []byte) ([]byte, error) {
 	}
 	return result, nil
 }
+
+func stringifyResponseHeader(header http.Header) (string, error) {
+	headerJSON, err := json.Marshal(header)
+	return string(headerJSON), err
+}
```

Comment:
A response header would basically be stored as a string representation of a JSON-ified response header.